src/components/RichEditor/index.jsx

```diff
@@ -103,8 +128,9 @@ class RichEditor extends React.Component {
               content_style: contentStyle,
               default_link_target: '_blank',
             }}
-            onChange={this.updateCharCount}
-            onKeyUp={this.updateCharCount}
+            onChange={this.handleOnChange}
+            onEditorChange={this.handleEditorChange}
```

Comment:
Can you provide a "noobs" explanation for when `onChange` and `onEditorChange` will exactly be triggered?

Reply:
OnEditorChange is for all things editor-related (content changes, copy/paste, undo/redo) while onChange is on the DOM level (focus, blur, etc.). TinyMCE renders as an iframe, and to have better control over how the content is managed, it has a separate event for Editor, to distinguish from DOM events.